build.yaml

```diff
@@ -8,6 +8,11 @@ build_from:
 codenotary:
   signer: notary@home-assistant.io
   base_image: notary@home-assistant.io
+cosign:
+  base_issuer: https://token.actions.githubusercontent.com
```

Comment:
Suggestion/idea: Should we default to this one? Assuming this will be the same in almost all cases.

Reply:
work for me